src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue8461.cs

```diff
@@ -1,35 +1,44 @@
-﻿using NUnit.Framework;
+﻿#if TEST_FAILS_ON_CATALYST // Using SwipeLeftToRight leads to exception of type 'OpenQA.Selenium.WebDriverException'. "Only pointer type 'mouse' is supported.
+using NUnit.Framework;
 using UITest.Appium;
 using UITest.Core;
 
 namespace Microsoft.Maui.TestCases.Tests.Issues;
 
 public class Issue8461 : _IssuesUITest
 {
+	const string ButtonId = "PageButtonId";
+	const string LayoutId = "LayoutId";
+	const string InstructionsLabel = "InstructionsLabel";
+	
 	public Issue8461(TestDevice testDevice) : base(testDevice)
 	{
 	}
 
-	public override string Issue => "[Bug] [iOS] [Shell] \"Nav Stack consistency error\"";
-
-	//[Test]
-	//[Category(UITestCategories.Navigation)]
-	//[FailsOnIOS]
-	//public void ShellSwipeToDismiss()
-	//{
-	//	var pushButton = App.WaitForElement(ButtonId);
-	//	Assert.AreEqual(1, pushButton.Length);
-
-	//	App.Tap(ButtonId);
-
-	//	var page2Layout = App.WaitForElement(LayoutId);
-	//	Assert.AreEqual(1, page2Layout.Length);
-	//	// Swipe in from left across 1/2 of screen width
-	//	App.SwipeLeftToRight(LayoutId, 0.99, 500, false);
-	//	// Swipe in from left across full screen width
-	//	App.SwipeLeftToRight(0.99, 500);
-
-	//	pushButton = App.WaitForElement(ButtonId);
-	//	Assert.AreEqual(1, pushButton.Length);
-	//}
-}
\ No newline at end of file
+	public override string Issue => "[Bug] [iOS] [Shell] Nav Stack consistency error";
+
+	[Test]
+	[Category(UITestCategories.Navigation)]
+
+	public void ShellSwipeToDismiss()
```

Comment:
This test is failing on Android and iOS:
<img width="1153" alt="image" src="https://github.com/user-attachments/assets/1226ee5d-d527-47c3-9f25-327d8c62e33f">

Reply:
Resolved the committed the changes
![image](https://github.com/user-attachments/assets/211c15ae-c1c6-42ee-a4da-4cdd5b44d482)